docs/messaging.rst

```diff
@@ -994,12 +998,48 @@ and expanded for JSON data::
       }
     }
 
+
+The ``transient`` dict contains runtime metadata that should not be persisted to
+document formats and is fully optional. The only transient key currently defined in Jupyter is
+``display_id``::
+
+    transient = {
+        'display_id': 'abcd'
+    }
+
 .. versionchanged:: 5.0
 
     `application/json` data should be unpacked JSON data,
     not double-serialized as a JSON string.
 
+Update Display Data
+-------------------
 
+Displays can now be named with a `display_id`` within the ``transient`` field of
+``display_data`` or ``execute_result``.
+
+When a ``display_id`` is specified for a display, it can be updated later
+with an ``update_display_data`` message. This message has the same format as `display_data`_
```

Comment:
Double check the '`' and triple backticks in this line and the next. Looks like some typos.